_sass/color_schemes/memo.scss

```diff
@@ -1,3 +1,58 @@
 // Change default font family
 $body-font-family: "Inter", system-ui, -apple-system, blinkmacsystemfont,
   "Segoe UI", roboto, "Helvetica Neue", arial, sans-serif;
+
+  // Typography
+
+  $body-font-family: "Inter";
```

Comment:
There is another definition for `$body-font-family` on line 2 that seems more sturdy. I think we should remove this one.

Reply:
That's a mistake. I will fix it on my next commit.